temporal/t.remove/t.remove.py

```diff
@@ -136,46 +147,51 @@ def main():
                 # We may have multiple layer for a single map, hence we need
                 # to avoid multiple deletation of the same map,
                 # but the database entries are still present and must be removed
-                if map.get_name() not in name_list:
-                    name_list.append(str(map.get_name()))
+                if clean:
+                    if map.get_name() not in name_list:
+                        name_list.append(str(map.get_name()))
+                elif not force:
+                    grass.message(_("  - %s" % map.get_name()))
+                    continue
                 map_statement += map.delete(dbif=dbif, execute=False)
 
                 count += 1
                 # Delete every 100 maps
                 if count % 100 == 0:
                     dbif.execute_transaction(map_statement)
-                    if type == "strds":
-                        remove(type="raster", name=name_list, run_=True)
-                    if type == "stvds":
-                        remove(type="vector", name=name_list, run_=True)
-                    if type == "str3ds":
-                        remove(type="raster_3d", name=name_list, run_=True)
+                    if clean:
+                        if type == "strds":
+                            remove(type="raster", name=name_list, run_=True)
+                        if type == "stvds":
+                            remove(type="vector", name=name_list, run_=True)
+                        if type == "str3ds":
+                            remove(type="raster_3d", name=name_list, run_=True)
                     map_statement = ""
                     name_list = []
 
             if map_statement:
                 dbif.execute_transaction(map_statement)
-            if name_list:
+            if clean and name_list:
                 if type == "strds":
                     remove(type="raster", name=name_list, run_=True)
                 if type == "stvds":
                     remove(type="vector", name=name_list, run_=True)
                 if type == "str3ds":
                     remove(type="raster_3d", name=name_list, run_=True)
-        else:
-            grass.message(
-                _(
-                    "Note: registered maps themselves have not been removed, only the %s"
-                    % type
-                )
-            )
+        if force or clean:
+            statement += sp.delete(dbif=dbif, execute=False)
 
-        statement += sp.delete(dbif=dbif, execute=False)
-
-    # Execute the collected SQL statenents
-    dbif.execute_transaction(statement)
-
-    dbif.close()
+    if not force and not clean:
+        grass.message(
+            _(
+                "Nothing removed. You must use the force flag (-{flag}) to actually "
+                "remove them. Exiting.".format(flag="f")
```

Comment:
```suggestion
                "remove them.".format(flag="f")
```

Reply:
The string is coming from g.remove, we should think to update also there

Reply:
yes, indeed, i'd remove the "would" and "Exiting" from g.remove